test/runtests.jl

```diff
@@ -25,7 +24,6 @@ ArrayType = Array
 @info ArrayType
 
 @testset "SurfaceFluxes - Recovery Profiles" begin
-    FloatType = Float32
```

Comment:
How is this allowed to be deleted?

Reply:
`FloatType` gets set on line 14, but I can move it back into the @testset

Reply:
Ah, I see. Removing it is fine.